pkg/machines/xml-helpers/xml2js.js

```diff
@@ -0,0 +1,363 @@
+// This is copied from https://github.com/nashwaan/xml-js/blob/master/lib/xml2js.js
+
+import * as sax from 'sax';
```

Comment:
I suppose there was some misunderstanding. If we have to copy that much code, then let's just use xml-js; having verbatim code copies is worse. It's not literally the dependency declaration in package.json that is the problem, but having to add (externally or internally) a lot of new code.

My question back then was about whether we really need to do this XML → JS → XML conversion dance in the first place, as it's lossy and at least structurally not really necessary. If manipulating XML is too unwieldy, then it seems better to me to introduce some helper functions for *that*.

Reply:
We do need to change attributes in existing domains XML. That's why I thought that having the XMLs as object will result in cleaner code. But of course it's not the only way to do it.

So, I will rewrite this PR and instead of having all these xmljs helpers I will parse directly the DomParser element and save it with the Domain description object. 
I trust you that it might not be that bad in the end to just directly parse the DOMPrser.